.github/dependabot.yml

```diff
@@ -22,6 +22,30 @@ updates:
       interval: daily
     target-branch: dev
 
+#  - package-ecosystem: terraform
+#    directory: '/terraform-iac/dev/app'
+#    schedule:
+#      interval: daily
+#    target-branch: dev
+
+#  - package-ecosystem: terraform
+#    directory: '/terraform-iac/dev/setup'
+#    schedule:
+#      interval: daily
+#    target-branch: dev
+
+  - package-ecosystem: terraform
+    directory: '/terraform-iac/modules/app'
+    schedule:
+      interval: daily
```

Comment:
(nit) is there a reason this is daily? I've been doing weekly just so that I don't get overwhelmed with updates.

Reply:
All(?) of the Terraform modules we're using are ones we've written. They don't update very often. 🤷‍♂️